crates/bevy_ecs/src/change_detection.rs

```diff
@@ -100,7 +105,7 @@ macro_rules! change_detection_impl {
 }
 
 macro_rules! impl_into_inner {
-    ($name:ident < $( $generics:tt ),+ >, $target:ty, $($traits:ident)?) => {
+    ($name:ident < $( $generics:tt ),+ >, $target:ty, $($traits:tt)?) => {
```

Comment:
`$traits` should be renamed has it can now handle more than just traits. `$bindings`?